ceph.spec.in

```diff
@@ -403,7 +402,6 @@ Summary:	OCF-compliant resource agents for Ceph daemons
 %if 0%{?suse_version}
 Group:		System/Filesystems
 %endif
-License:	LGPL-2.0
```

Comment:
okay, i *guess* probably we can use a single license at the moment until someone justifies that it's necessary to license a subpackage with a difference license. but since IANAL and i am far from an expert in this area, so cannot be 100% sure about this change.